tools/local_development/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM fedora:27
+
+MAINTAINER Bartek Kowalczyk <bkowalczyyk@gmail.com>
```

Comment:
Let's change to the smf-dev@googlegroups.com for the email

Reply:
done

Reply:
cool, pls make sure to squash your commits.